include/drjit/packet_neon.h

```diff
@@ -0,0 +1,1259 @@
+/*
+    drjit/packet_neon.h -- Packet arrays, ARM NEON specialization
+
+    Dr.Jit is a C++ template library for efficient vectorization and
+    differentiation of numerical kernels on modern processor architectures.
+
+    Copyright (c) 2021 Wenzel Jakob <wenzel.jakob@epfl.ch>
+
+    All rights reserved. Use of this source code is governed by a BSD-style
+    license that can be found in the LICENSE file.
+*/
+
+#pragma once
+
+#include <drjit/half.h>
+
+NAMESPACE_BEGIN(drjit)
+DRJIT_PACKET_DECLARE(16)
```

Comment:
This change was needed to build for android32:

```suggestion	
DRJIT_PACKET_DECLARE_COND(16, enable_if_t<(std::is_same_v<Type, float>)>)
DRJIT_PACKET_DECLARE_COND(16, enable_if_int32_t<Type>)
#if defined(ENOKI_ARM_64)
DRJIT_PACKET_DECLARE_COND(16, enable_if_t<(std::is_same_v<Type, double>)>)
DRJIT_PACKET_DECLARE_COND(16, enable_if_int64_t<Type>)
#endif
```